contracts/lockdrop/src/contract.rs

```diff
@@ -1086,8 +1086,14 @@ pub fn handle_claim_rewards_and_unlock_for_lockup(
                     }
                     .to_cosmos_msg(&env)?,
                 );
+            } else if user_info.astro_transferred {
```

Comment:
We should also check that users aren't going to withdraw astroport lp tokens, because it would be impossible to withdraw lp tokens if they were staked to generator and no rewards are available now

Reply:
yes very true, added the check